apps/juicebox_web/assets/js/videos/video.scss

```diff
@@ -1,13 +1,41 @@
-.video-wrapper {
+.video-wrapper,
+.video-container {
   position: relative;
-  padding-bottom: 56.25%; /* 16:9 */
-  padding-top: 25px;
-  height: 0;
 }
+
+.video-expanded.video-present {
+  .video-wrapper {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
+    padding-top: 25px;
+    height: 0;
+  }
+}
+
 .video-wrapper iframe {
   position: absolute;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
 }
+
+.video-state-button {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+}
+
+.video-title {
+  width: 100%;
```

Comment:
Properties should be ordered bottom, display, width, height, padding, font-size, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)